.goreleaser.yml

```diff
@@ -23,6 +24,38 @@ builds:
           goarch: 386
         - goos: windows
           goarch: 386
+- id: signatory-cli
+  binary: signatory-cli
+  env:
+    - CGO_ENABLED=0
+  main: ./cmd/signatory-cli/main.go
+  ldflags:
+    - -X github.com/ecadlabs/signatory/pkg/metrics.GitRevision={{.Version}}
+    - -X github.com/ecadlabs/signatory/pkg/metrics.GitBranch={{.Version}}
+  goos:
+    - freebsd
+    - linux
+    - windows
+    - darwin
+  goarch:
+    - amd64
+    - arm
+  ignore:
+    - goos: darwin
+      goarch: 386
+    - goos: windows
+      goarch: 386
+dockers:
+  - 
+    binaries:
+    - signatory
+    - signatory-cli
+    builds:
+    - signatory
+    - signatory-cli
+    image_templates:
+    - "technomad21c/signatory:latest"
```

Comment:
Please update to `ecadlabs/signatory:latest`

Reply:
updated. Thanks.